doc_src/cmds/string-split.rst

```diff
@@ -49,9 +51,8 @@ Examples
     b
     c
 
-    >_ string split -f1,3 '' abc
+    >_ string split --allow-empty -f1,5 '' abc
```

Comment:
I think the old example was nicer, because it shows the common use case.

Reply:
Yeah I wanted to also show `--allow-empty` in case it's behaviour wasn't obvious from the description and figured I'd merge it into that one instead of making another example.

Reply:
Maybe do `-f1,3,5` here to have both?

Reply:
Done